app/scripts/background.js

```diff
@@ -1,3 +1,5 @@
+/* global chrome */
```

Comment:
Could we use the Mozilla polyfill here rather than using the `chrome` global directly?

Reply:
The polyfill we are using `browser` does not seems to support this specific api `chrome.tabs`

Reply:
I'm pretty sure it does. We already use it, in `app/scripts/platform/extension.js`

Reply:
Using the polyfill LGTM. updated here https://github.com/MetaMask/metamask-extension/pull/16075/commits/00471464812bc33f37510624743ec6698798d590